### ISSUE TITLE
Change capitalization throughout for consistency

### ISSUE THREAD

**Issue by danid123** [GSA/openacr#222]
Following 18f's guidelines:
- Sentence capitalization for titles and headings, ex. Structure of this tool
- Title caps for labels that are nouns, ex. Company Name
- Sentence capitalization for labels that are actions, ex. Add section

Updates to be made:
- Currently, text field labels are already consistent.
- Page headings are currently using title case but should be using sentence case
- Section headings are mostly using title case but should be using sentence case

**Comment by mgifford** [comment]
I am terrible at this. We need to look through and find more instances where it hasn't been done consistently.

**Comment by mgifford** [comment]
Ok, so looking at:
 https://gsa.github.io/openacr-editor/chapter/support_documentation_and_services

What do we do when we are referring to documents that use Title Case? 

These are all based on Access Board documents:

https://www.access-board.gov/ict/

I think that if we're referring to an external document that is in Title Case we have to leave it that way. All of these apply.

**Comment by bruce-usab** [comment]
Just my own prejudices here.  At USAB, our practice is to always capitalize EXCEPTION.  Please let that data point inform how closely you wish to follow our model...

WRT title case versus sentence case, yes please be consistent.

18F might have there own style guide.   From a search just now, it looks to me like USWDS v1 strongly advocated for sentence case over title case, but dropped that for v2.  @danid123 do you happen to have a public-facing URL arguing for sentence case on headings?

From what I found at [APA style guide](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) there seems to be support for title case on headings.

It will be interesting to see if this comes up in the context of guidance which might follow WCAG3.

**Comment by danid123** [comment]
Title case is harder to read for long sentence-length titles because reader's eye does a lot of jumping up and down over the letters and cognitively the reader processes each word rather than the phrase.  Sentence case is also perceived as friendlier which is helpful for a site with a lot of technical terms.

I couldn't find the specific line in the 18F guidelines but other civic tech resources collected some info on this topic. 

Readability guidelines UK: 
https://readabilityguidelines.co.uk/grammar-points/capital-letters/

**Comment by bruce-usab** [comment]
> With EXCEPTION you don't mean like here ... But with sub-headings such as used here...

Yes, exactly.  USAB uses All Caps (sometimes) as emphasis for consistency with stylistic choices from the pre-wordprocessor days.  An argument could be made for those should be a `strong em` or even `h`.  That was meant _not_ as example that OpenACR should follow, but as a caution not to give too much weight for USAB preference for Title Case over Sentence Case.

> Ultimately the GSA may need to decide on this, but I'm fine following existing examples in .gov sites where we can. Working towards consistency.

So very good!

> 18F has this guide for content

Okay, that is unambiguous, thanks for that!

**Comment by melindaburgess** [comment]
I also heartily agree that sentence case is a better choice, especially in government applications where there are a lot of acronyms and proper nouns that must be capitalized. Using sentence case for headings and titles makes it easier to distinguish what is a proper noun.

E.g.
Using the OpenACR tool at the General Services Administration

**Comment by mgifford** [comment]
Are there more places where we should do this? Can we move this to done?
